src/test/java/org/jenkinsci/plugins/workflow/steps/EnvStepTest.java

```diff
@@ -105,6 +135,28 @@
         });
     }
 
+    @Test public void mapParallel() {
```

Comment:
Again, unnecessary, adequately covered by existing tests; all that is changing is the syntax.